source/nodejs/adaptivecards-designer/cdn.html

```diff
@@ -25,14 +25,18 @@
 			height: 100vh;
 		}
 	</style>
+	<script src="https://unpkg.com/adaptivecards@latest/dist/adaptivecards.min.js"></script>
+	<script src="https://unpkg.com/adaptive-expressions@4.11.0-dev.20201027.ca35ce4/lib/browser.js"></script>
```

Comment:
same commment above about hard coded version

Reply:
changed to rc